From 8bccf237209a5e4ecc02b45e51cf1fd1e42cf2a0 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Wed, 9 Mar 2016 12:02:30 -0600
Subject: [PATCH 0531/1164] CDH-38166: HIVE-13244: Get
 'test-query-redactor.json' file on
 TestQueryRedactor.java from the resources
 classpath

Change-Id: Icd7e527fa18b6dcc62df522c14edfe07e240d750
---
 .../hadoop/hive/ql/hooks/TestQueryRedactor.java    |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ql/src/test/org/cloudera/hadoop/hive/ql/hooks/TestQueryRedactor.java b/ql/src/test/org/cloudera/hadoop/hive/ql/hooks/TestQueryRedactor.java
index e2516a2..57a8946 100644
--- a/ql/src/test/org/cloudera/hadoop/hive/ql/hooks/TestQueryRedactor.java
+++ b/ql/src/test/org/cloudera/hadoop/hive/ql/hooks/TestQueryRedactor.java
@@ -23,6 +23,7 @@
 import org.apache.hadoop.hive.conf.HiveConf;
 import org.apache.hadoop.hive.ql.Driver;
 import org.apache.hadoop.hive.ql.session.SessionState;
+import org.apache.hive.common.util.HiveTestUtils;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.AfterClass;
@@ -56,7 +57,7 @@ public void testQueryRedactor() throws Exception {
       QueryRedactor.class.getName());
     String hiveRoot = System.getProperty("hive.root");
     assertNotNull("Hive root cannot be null", hiveRoot);
-    conf.set("hive.query.redaction.rules", hiveRoot + "/ql/src/test/resources/test-query-redactor.json");
+    conf.set("hive.query.redaction.rules", HiveTestUtils.getFileFromClasspath("test-query-redactor.json"));
     Driver driver;
     int ret;
     driver = createDriver(conf);
-- 
1.7.9.5

